src/dag.ts

```diff
@@ -11,7 +12,21 @@ export interface DagCommandInterface {
 
   modelrun(modelName: string, inputs: string[], outputs: string[]): DagCommandInterface;
 
+  modelexecute(modelName: string, inputs: string[], outputs: string[], timeout?: number): DagCommandInterface;
+
```

Comment:
can you deprecate `modelrun` and `scriptrun` here?

Reply:
done